applications/asset_tracker_v2/src/cloud/cloud_codec/cloud_codec.h

```diff
@@ -18,6 +18,12 @@
 #include <net/wifi_location_common.h>
 #include <nrf_modem_gnss.h>
 
+#if defined(CONFIG_LWM2M)
+#include <zephyr/net/lwm2m.h>
+#else
+#include "lwm2m/lwm2m_dummy.h"
```

Comment:
Would love to avoid inclusion of this dummy file. But, I don't see anyway around.

Reply:
Yeah, that was a bit of a headache, but didn't find a better solution.